src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -53,23 +57,41 @@ export const ViewMyNftsAsset = ({
 
   const assetMediaType = Card.useAssetMediaType(asset)
 
+  const isDisabled = asset.asset_contract.tokenType === 'ERC1155' || (isSellMode && asset.susFlag)
+  const disabledTooltipText =
+    asset.asset_contract.tokenType === 'ERC1155' ? 'ERC-1155 support coming soon' : 'Blocked from trading'
+
   return (
     <Card.Container
       asset={asset}
       selected={isSelected}
       addAssetToBag={() => handleSelect(false)}
       removeAssetFromBag={() => handleSelect(true)}
       onClick={onCardClick}
+      isDisabled={isDisabled}
     >
-      <Card.ImageContainer>
-        {assetMediaType === AssetMediaType.Image ? (
-          <Card.Image />
-        ) : assetMediaType === AssetMediaType.Video ? (
-          <Card.Video shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        ) : (
-          <Card.Audio shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        )}
-      </Card.ImageContainer>
+      <MouseoverTooltip
+        text={
+          <Box as="span" className={bodySmall} style={{ color: themeVars.colors.textPrimary }}>
+            <Trans>{disabledTooltipText}</Trans>{' '}
+          </Box>
+        }
+        placement="bottom"
+        offsetX={0}
+        offsetY={-100}
+        style={{ display: 'block' }}
+        disableHover={!isDisabled}
+      >
+        <Card.ImageContainer>
```

Comment:
Could you simplify this? The nested tertiary is confusing

Reply:
sounds good changed it to a switch statement

Reply:
beautiful!